docs/source/fitting/fitfunctions/MultiDomainFunction.rst

```diff
@@ -0,0 +1,86 @@
+.. _func-MultiDomainFunction:
+
+=======================
+Multi Domain Function
+=======================
```

Comment:
I haven't checked the docs build yet, but does this underline need to line up with the text for it to work?

Reply:
Not sure to be honest, but I lined it up just to make sure.